system/src/Grav/Common/GPM/GPM.php

```diff
@@ -139,6 +139,32 @@ public function getInstalledPlugins()
         return $this->installed['plugins'];
     }
 
+
+    /**
+     * Returns the plugin's enabled state
+     *
+     * @param  string $slug
+     * @return bool True if the Plugin is Enabled. False if manually set to enable:false. Null otherwise.
+     */
+    public function isPluginEnabled($slug)
+    {
+        $result = null;
+        $grav = new Grav();
+        $config = $grav::instance()['config']['plugins'];
+        $installed = $this->isPluginInstalled($slug);
+
+        $enabled = isset($config[$slug]) && ((bool)$config[$slug]['enabled'] == true);
+        $disabled = isset($config[$slug]['enabled']) && ((bool)$config[$slug]['enabled'] === false);
```

Comment:
Umm, aren't these two lines the same?

Here's better code:
```
return (bool)($config[$slug]['enabled'] ?? false);
```

Reply:
I think there are technically 3 states:
1. config exists and is set to enabled
2. config exists and is set to disabled
3. config doesn't exist (grav treats this as disabled)

I was attempting to take into consideration the tricky difference between #2 and #3 cases.